spec/controllers/admin/content_controller_spec.rb

```diff
@@ -3,6 +3,40 @@
 describe Admin::ContentController do
   render_views
 
+  describe '#merge' do
+    before do
+      Factory(:blog)
+    end
+
+    let!(:article) do
+      Factory(:article)
+    end
+
+    let!(:another_article) do
+      Factory(:article)
+    end
+
+    let(:article_params) do
+      { "merge_with"=>another_article.id,
+        "id"=>article.id
+      }
+    end
+
+    it 'should not merge when the user is not an admin' do
+      original_count = Article.count
```

Comment:
Since this variable is used in both tests, it might make sense to pull it out into a let or the before